organization/annual-reports/annual-report-2019-2020.md

```diff
@@ -0,0 +1,486 @@
+
+### Our people
+
+**In 2019, our initial team included:**
+
+* **Boris van Hoytema** (Chief Executive, co-founder)
+* **Ben Cerveny** (President, co-founder)
+* **Elena Findley-de Regt** (Communications Coordinator)
+* **Claus Mullie** (Codebase Stewardship Coordinator)
+* **Mirjam van Tiel** (Operations Coordinator until December)
+
+with **Willemijn van Bekkum** providing pro-bono legal and human resources support.
+
+**They were later joined by:**
+
+* **Eric Herman** (Lead Codebase Steward for Quality)
+* **Jan Ainali** (Codebase Steward for Community)
+
+with **Floris Deerenberg** and **Ryan Conlan** providing ad hoc illustration, design and communication support.
+
+**In 2020, we welcomed:**
+
+* **Deborah Meibergen** (Operations Coordinator)
+* **Alba Roza** (Codebase Steward for Community)
+* **Laura Scheske** (Membership Developer)
+* **Felix Faassen** (Codebase Steward for Product)
+
+with **Rob Schouten** providing financial expertise together with our bookkeepers from Get Monkeys.
+
+### Our board of directors
+
+* **Arnout Schuijff** (Treasurer)
+* **Ben Cerveny** (Chairman)
+* **Boris van Hoytema** (Secretary)
+* **Pieter van der Does**
+
+Board members do not receive any compensation, and did not incur any expenses for their activities.
+
+![Finances section banner](https://raw.githubusercontent.com/publiccodenet/about/annual-report-2019-2020/organization/annual-reports/images/ar-19-20-b04.png)
+
+## Our Finances
+
+This section provides an overview of the financials that accompanies this annual report.
+
+[See the full financial report](https://about.publiccode.net/organization/annual-reports/financial-report-2019-2020).
+
+In its startup phase, this organization was dependent on philanthropic donations. It has been the aim of the Foundation to eventually be self-supporting through the contributions of public organizations.
+
+Generous philanthropic donations have allowed us to expand our stewardship operations, conduct research and development, and raise awareness around public code. Our donors pledged a total of €2,983,857 in 2019 and 2020. To prevent undue influence, all donations are un-earmarked and go into our general operating budget, in compliance with our status as a Dutch public benefit organization (ANBI).
+
+During this startup phase, we try to look two years ahead with our financial planning. The organization has low financial risk: it is not capital intensive (no large capital expenditures) and has no loans with regard to the organization’s assets.
+
+The current financial situation gives the Foundation for Public Code financial security until April 2022.
+
+**Our largest cost by far (really, our biggest investment) is the amazing people who make our work possible. This includes their salaries, wage taxes, insurances and expenses.**
+
+![80% people](https://raw.githubusercontent.com/publiccodenet/about/annual-report-2019-2020/organization/annual-reports/images/80per-people.png)
```

Comment:
I noticed this graphic only works on white, a potential risk in future reuse, but not an issue for now.

Reply:
should we create an issue to ensure it's tackled in the future?

Reply:
Thanks, good spot. @demkodo, please tag @angelplasma in the issue?